src/man/sssd.8.xml

```diff
@@ -90,6 +90,41 @@
                         log files are stored in <filename>/var/log/sssd</filename> and
                         there are separate log files for every SSSD service and domain.
                     </para>
+                    <para>
+                        This option is deprecated. It is replaced by
+                        <option>--logger=files</option>.
+                    </para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term>
+                    <option>--logger=</option><replaceable>value</replaceable>
+                </term>
+                <listitem>
+                    <para>
+                        Send messages to different place. This option overrides
```

Comment:
`Location where SSSD will send log messages to` sounds a bit better here imo

Reply:
Yes,  it sounds better :-)